src/models/user.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const userSchema = Schema({
+  userName: {
+    type: String,
+    required: [true, 'Username is required'],
+  },
+  firstName: {
+    type: String,
+    required: [true, 'First name is required'],
+  },
+  lastName: {
+    type: String,
+    required: [true, 'Last name is required'],
+  },
+  email: {
+    type: String,
+    required: [true, 'Email is required'],
```

Comment:
"email" should be unique as well, in order not to have multiple users with the same email. 
It could be avoided by checking all users' emails but making it unique and letting mongoose do the work would be easier.

Reply:
I changed it. Thanks for the feedback.